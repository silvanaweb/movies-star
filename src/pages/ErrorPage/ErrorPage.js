import React from "react";
import "./ErrorPage.css";

const ErrorPage = ({ errorMessage }) => (
  <div className="ErrorPage">
    <h1>{errorMessage}</h1>
  </div>
);

export { ErrorPage };
