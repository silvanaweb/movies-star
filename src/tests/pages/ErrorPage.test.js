import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";
import React from "react";
import { ErrorPage } from "../../pages/ErrorPage/ErrorPage";

let wrapper;

beforeEach(() => {
	wrapper = shallow(<ErrorPage errorMessage={""} />);
});

test("should render ErrorPage correctly", () => {
	expect(toJSON(wrapper)).toMatchSnapshot();
});
test("should render ErrorPage wit message correctly", () => {
	wrapper.setProps({
		errorMessage: "An error occurred",
	});
	expect(toJSON(wrapper)).toMatchSnapshot();
});
