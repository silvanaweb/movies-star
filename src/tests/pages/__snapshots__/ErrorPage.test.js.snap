// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ErrorPage correctly 1`] = `
<div
  className="ErrorPage"
>
  <h1 />
</div>
`;

exports[`should render ErrorPage wit message correctly 1`] = `
<div
  className="ErrorPage"
>
  <h1>
    An error occurred
  </h1>
</div>
`;
